README.md

```diff
@@ -76,10 +78,10 @@ An example _software_ domain is provided in the same directory as this styleguid
 A domain **must** use the following file structure:
 
 ```
-apis.py - Public functions and access points.
+apis.py - Public functions and access points, presentation logic.
 interfaces.py - Integrations with other domains or external services.
-models.py - Object models and storage.
-services.py - Business and functional logic.
+models.py - Object models and storage, simple information logic.
+services.py - coordination and functional logic.
```

Comment:
How do you feel about replacing `functional` with `transactional`? Because functional is such an overloaded term...